analyzers/python/integration_test.go

```diff
@@ -0,0 +1,133 @@
+package python_test
+
+import (
+	"flag"
+	"os"
+	"path/filepath"
+	"testing"
+
+	"github.com/apex/log"
+
+	"github.com/fossas/fossa-cli/analyzers"
+	"github.com/fossas/fossa-cli/exec"
+	"github.com/fossas/fossa-cli/module"
+	"github.com/fossas/fossa-cli/pkg"
+	"github.com/fossas/fossa-cli/testing/fixtures"
+	"github.com/stretchr/testify/assert"
+)
+
+var pythonAnalyzerFixtureDir = filepath.Join(fixtures.Directory(), "python", "analyzer")
```

Comment:
Is it necessary to namespace the fixtures? I don't think we currently do, and we have yet to run in to a name collision.

Reply:
The idea here is that we can cache on a particular language/ecosystem instead of whether or not there has been any integration test run. Additionally, it's a nice organization when inspecting issues that pop up